packages/pf3-component-mapper/demo/demo-schemas/sandbox.js

```diff
@@ -405,9 +405,10 @@ const output = {
                   component: components.DATE_PICKER,
                   isClearable: true,
                   disabledDays: [
+                    'today',
```

Comment:
What this line actually means? Is it that you can't select today in the picker? Would it disable any date befor today if I write `before: 'today'` below?

Reply:
http://react-day-picker.js.org/api/DayPicker#disabledDays

Reply:
We are using this component. There is a link in the componentApi docs for date time picker

Reply:
Yeah, this doesn't really answer my question, but I tested it and it works:
```js
                  disabledDays: [{
                    after: 'today'
                  }]
```
This doesn't allow me to select any date in the future and that's the thing I want the most from this PR :wink: